resources/contents/blog/en/user-experience-names.md

```diff
@@ -0,0 +1,74 @@
+
+### Multiple family names
+
+There are other cultures — such as Spanish speaking cultures — that do not only have one lastname, but two. María-Jose Carreño Quiñones might be the daughter of Antonio Carreño Rodríguez and María Quiñones Marqués and you would direct her as Señora Carreño not Señora Quiñones.
+
+In Brazil you even have more than two lastnames, taken from multiple generations. How would you direct José Paulo Bezerra Maciel Júnior?
+
+### Multiple firstnames
+
+During my time in Germany I met someone whose firstname was Anna Lena. But she always felt uncomfortable when somebody called her will the full firstname. She wanted to be directed with the second part, just as Lena
+
+---
+
+As you see, different cultures and different people can have different names. Do you want to force them all to choose a firstname and a lastname just for the sake of your database columns?
+
+## How to ask for names
+
+But how would you ask for your users names? The first thing you need to answer yourself: Do you actually need the name? In the time of GDPR and the awarenesses of data protection, spend some time of you really need the name of your user. To make the user signing up for your newsletter, you only need an email address, to make him covert to your application, an email address or a mobile number is enough.
+
+Only for really official things like payments, you have to ask for names and there you can just take the same approach as the banking industry does. You won't hurt the user at that point because the banks already did.
+
+### Ask for fullname and a displayname
+
+In your application, it is perfectly fine to ask the user on how they want to be directed. Ask for a displayname. They can then choose themselves if they prefer the polite way, or if just their firstname is fine.
+
+And for invoices and less personal situation just ask them for their fullnames. It doesn't matter that you have their firstname and lastnames there since you are going to combine them anyway.
+
+### Choose appropriate label names
+
+If you still need to ask for a firstname and a lastname, make sure that you use the correct terms. Instead of just asking for first-/lastnames choose something like "family name" and "given name".
+
+Also prefer using the term "birth name" instead of "maiden name" since it's just not always the case that women take over their husbands name.
+
+### Other things to consider
+
+Remember that names do not only contain letters, but also hyphens, spaces and other special characters. It is always possible that a name only consists of one letter and at the same time a name can also have more that just 12 characters. And always make sure that you indicate what you expect. Nothing is worst than failing in submitting a form because your own name was not accepted.
```

Comment:
```suggestion
Remember that names do not only contain letters, but also hyphens, spaces, and other special characters. It is always possible that a name only consists of one letter and at the same time a name can also have more than just 12 characters. And always make sure that you indicate what you expect. Nothing is worse than failing in submitting a form because your own name was not accepted. 
```